src/hotplug_generic.c

```diff
@@ -65,3 +66,28 @@ void HPReCheckSerialReaders(void)
 }
 
 #endif
+
```

Comment:
Why not use something like:
```C
char * HPGetenv(const char *name)
{
#ifdef HAVE_SECURE_GETENV
    return secure_getenv(name);
#else 
    if (issetugid())
        return NULL;
    return getenv(name);
#endif
}
```

And add a test for `secure_getenv()` function availability in `configure.ac`?